### PR TITLE
Add uninstaller

### PR DESCRIPTION
**Pull request by gerardroche**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -11,26 +11,25 @@ fi
 echo "Installing php-build in $PREFIX"
 
 BIN_DIR="$PREFIX/bin"
+SHARE_DIR="$PREFIX/share"
 MAN_DIR="$PREFIX/share/man"
-TMP_DIR="/tmp/php-build"
 
-echo -n "  - Creating Directories..."
+echo -n "  - creating directories..."
```
Comment (GrahamCampbell) [inline install.sh]: ?

File: install.sh [hunk 2]
```diff
@@ -11,26 +11,25 @@ fi
...
 [ ! -d "$BIN_DIR" ]          && mkdir -p "$BIN_DIR"
 [ ! -d "$MAN_DIR" ]          && mkdir -p "$MAN_DIR"
+[ ! -d "$SHARE_DIR" ]        && mkdir -p "$SHARE_DIR"
 [ ! -d "$MAN_DIR/man1" ]     && mkdir -p "$MAN_DIR/man1"
 [ ! -d "$MAN_DIR/man5" ]     && mkdir -p "$MAN_DIR/man5"
-[ ! -d "$TMP_DIR/packages" ] && mkdir -p "$TMP_DIR/packages"
-[ ! -d "$TMP_DIR/source" ]   && mkdir -p "$TMP_DIR/source"
 
-echo " Done."
+echo " done."
```
Comment (GrahamCampbell) [inline install.sh]: ?

File: install.sh [hunk 3]
```diff
@@ -11,26 +11,25 @@ fi
...
+echo -n "  - copying files..."
 
+cp -r "$DIR/"{bin,share} "$PREFIX/"
 cp "$DIR/man/php-build.1" "$MAN_DIR/man1/"
 cp "$DIR/man/php-build.5" "$MAN_DIR/man5/"
 
-echo " Done."
+echo " done."
+
+echo "Done."
```
Comment (GrahamCampbell) [inline install.sh]: Double done???

File: install.sh (hunk 2 above)
Comment (gerardroche) [inline install.sh]:
```
  - creating directories... Done.
  - creating directories... done.
```

The second looks nicer to me.

File: install.sh (hunk 1 above)
Comment (gerardroche) [inline install.sh]: lol I'm padantic. It looks nicer when printed.

File: install.sh (hunk 3 above)
Comment (gerardroche) [inline install.sh]:
```
Installing php-build in /usr/local
  - creating directories... done.
  - copying files... done.
Done.
```

**Comment by gerardroche** [comment]
All the commands could also be turn on verbose e.g. `cp -v`, `mkdir -pv`, etc. But I felt it was maybe a little too much output noise.

File: install.sh (hunk 3 above)
Comment (GrahamCampbell) [inline install.sh]: Right. :)

File: uninstall.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+BIN_DIR="$PREFIX/bin"
+SHARE_DIR="$PREFIX/share"
+MAN_DIR="$PREFIX/share/man"
+
+echo -n "  - removing files..."
+
+rm -f "$BIN_DIR/php-build"
+rm -f "$BIN_DIR/phpenv-install"
+rm -f "$BIN_DIR/rbenv-install"
```
Comment (rogeriopradoj) [inline uninstall.sh]: Could this line be wrong when user has installed https://github.com/sstephenson/rbenv?

Comment (gerardroche) [inline uninstall.sh]: `rbenv-install` is copied from php-build/bin/ on install. 

Looking at [rbenv](https://github.com/sstephenson/rbenv/tree/master/bin) I see `rbenv/bin/rbenv`. 

This is probably why the script is called `rbenv-install` and not `rbenv`

So it looks ok to me.

**Comment by rogeriopradoj** [comment]
Alright, tks!

**Comment by rogeriopradoj** [comment]
Thanks, @gerardroche !

**Pull request merged**